set.mm

```diff
@@ -486967,6 +487079,35 @@ its graph has a given second element (that is, function value).
     GDJCKADUGLMNUJUFUHUJUFUHOCUIFZUGUIJZOUKBCPQFULCGBSCGBRTUJUFUKUHULAUICUAAUIU
     GLUBMUCUD $.
 
+  ${
+    nn0difffzod.1 $e |- ( ph -> N e. ZZ ) $.
+    nn0difffzod.2 $e |- ( ph -> M e. ( NN0 \ ( 0 ..^ N ) ) ) $.
+    $( Membership in the nonnegative integers, except a half-open range
+       starting from zero, implies inequality.  (Contributed by Thierry Arnoux,
+       20-Feb-2025.) $)
+    nn0difffzod $p |- ( ph -> -. M < N ) $=
```

Comment:
We have M is greater-than-or-equal-to N, so there could still be equality

Maybe, "a nonnegative integer that is not in the half-open range from 0 to n is at least n".

Reply:
Changed in https://github.com/metamath/set.mm/pull/4663/commits/33dc4f338d18a4f3f69f8589eb18e8d55a2be41a